pkg/ibgu/ibgu.go

```diff
@@ -113,6 +113,32 @@ func (builder *IbguBuilder) WithClusterLabelSelectors(labels map[string]string)
 	return builder
 }
 
+// WithAutoRollbackOnFailure appends the AutoRollbackOnFailure to the ibuSpec.
```

Comment:
I'd add the InitMonitorTimeout related stuff to this comment too, but ...

Reply:
sure, added.